### PR TITLE
Backstage Plugin overview.

### PR DESCRIPTION
**Pull request by bigkevmcd**
Closes

**What changed?**
This provides a brief introduction to the Backstage Plugin for Flux.

It references the plugin README as the primary source for installation instructions.

**Why was this change made?**
To provide simple documentation on the Backstage plugin

**How was this change implemented?**
Documentation

**How did you validate the change?**

**Release notes**

**Documentation Changes**

### CODE REVIEW

File: website/docs/backstage/intro.mdx [hunk 1]
```diff
@@ -0,0 +1,135 @@
+---
+title: Backstage Plugin for Flux
+---
+
+Are you running [Backstage](https://backstage.io) and [Flux](https://fluxcd.io), do you want to expose the state of your Flux resources in your Backstage portal?
```
Comment (LappleApple) [inline website/docs/backstage/intro.mdx:5]:
```suggestion
Are you running [Backstage](https://backstage.io) and [Flux](https://fluxcd.io)? Do you want to expose the state of your Flux resources in your Backstage portal?
```

File: website/docs/backstage/intro.mdx [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+The `@weaveworksoss/backstage-plugin-flux` Backstage plugin provides a set of components that you can add to your existing Backstage app that will display the state of Flux resources.
```
Comment (LappleApple) [inline website/docs/backstage/intro.mdx:7]:
```suggestion
The `@weaveworksoss/backstage-plugin-flux` Backstage plugin provides a set of components that you can add to your existing Backstage app to display the state of Flux resources.
```

File: website/docs/backstage/intro.mdx [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+## Installation
+
+The full installation instructions are provided in the plugin [repository](https://github.com/weaveworks/weaveworks-backstage/tree/main/plugins/backstage-plugin-flux) but you will need the [Kubernetes plugin](https://backstage.io/docs/features/kubernetes/) installed and configured to access the clusters you want to query Flux resources from.
```
Comment (LappleApple) [inline website/docs/backstage/intro.mdx:11]:
```suggestion
We provide the full installation instructions in the plugin [repository](https://github.com/weaveworks/weaveworks-backstage/tree/main/plugins/backstage-plugin-flux). But first you will need to install the [Kubernetes plugin](https://backstage.io/docs/features/kubernetes/) and configure it to access the clusters you want to query Flux resources from.
```

File: website/docs/backstage/intro.mdx [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+# From your Backstage root directory
+yarn add --cwd packages/app @weaveworksoss/backstage-plugin-flux
+```
+
+Then add the components you want to your [EntityPage](https://backstage.io/docs/plugins/integrating-plugin-into-software-catalog/#import-your-plugin-and-embed-in-the-entities-page).
+
+Currently, the Backstage plugin provides the following components:
+
+- EntityFluxDeploymentsCard - shows a combined view of HelmReleases and Kustomizations
+- EntityFluxSourcesCard - shows a combined view of GitRepositories, OCIRepositories and HelmRepositories
```
Comment (LappleApple) [inline website/docs/backstage/intro.mdx:25]:
```suggestion
- EntityFluxSourcesCard - shows a combined view of GitRepositories, OCIRepositories, and HelmRepositories
```

File: website/docs/backstage/intro.mdx [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+- EntityFluxHelmReleasesCard
+- EntityFluxKustomizationsCard
+- EntityFluxGitRepositoriesCard
+- EntityFluxOCIRepositoriesCard
+- EntityFluxHelmRepositoriesCard
+
+For example, to add the `EntityFluxHelmReleasesCard` to your Entity home page for components with the `backstage.io/kubernetes-id` entity annotation.
```
Comment (LappleApple) [inline website/docs/backstage/intro.mdx:32]: This sentence feels unfinished? Specifically, "For example, to add...": the "to add" clause isn't closed.

File: website/docs/backstage/intro.mdx [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+      version: '6.3.6'
+      sourceRef:
+        kind: HelmRepository
+        name: podinfo
+        namespace: podinfo
+```
+
+![Entity Overview](img/helm-releases-in-overview.png)
+
+## Building a custom page with resources
```
Comment (LappleApple) [inline website/docs/backstage/intro.mdx:90]:
```suggestion
## Building a Custom Page with Resources
```

File: website/docs/backstage/intro.mdx [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+
+Instead of displaying the state on the overview page, it's possible to compose a page displaying the state of resources.
+
+For example, to add a page `/kustomizations` to your Entity for components with the `backstage.io/kubernetes-id` entity annotation.
```
Comment (LappleApple) [inline website/docs/backstage/intro.mdx:94]:
```suggestion
For example, to add a page `/kustomizations` to your Entity for components with the `backstage.io/kubernetes-id` entity annotation:
```

File: website/docs/backstage/intro.mdx [hunk 8]
```diff
@@ -0,0 +1,135 @@
...
+        <EntityFluxGitRepositoriesCard />
+      </Grid>
+    </Grid>
+  </EntityLayout.Route>
+);
+```
+
+![Custom Kustomizations Page](img/kustomizations-tab.png)
+
+## Connecting to WeaveGitOps
```
Comment (LappleApple) [inline website/docs/backstage/intro.mdx:121]:
```suggestion
## Connecting to Weave GitOps
```

File: website/docs/backstage/intro.mdx (hunk 4 above)
Comment (bigkevmcd) [reply website/docs/backstage/intro.mdx:25]: Not sure a comma is needed here?

Comment (LappleApple) [reply website/docs/backstage/intro.mdx:25]: It's a US English grammar norm, feel free to disregard.

File: website/docs/backstage/intro.mdx [hunk 9]
```diff
@@ -0,0 +1,135 @@
+---
+title: Backstage Plugin for Flux
+---
+
+Are you running [Backstage](https://backstage.io) and [Flux](https://fluxcd.io)? Do you want to expose the state of your Flux resources in your Backstage portal?
```
Comment (foot) [inline website/docs/backstage/intro.mdx:5]: 🔥

**Review by foot — APPROVED** [review]
🏅 LGTM

**Pull request merged**